tests/cassettes/test_cmip6_datacube/test_datacube_helper.yaml

```diff
@@ -0,0 +1,250 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Connection:
+      - close
+      Host:
+      - stac-extensions.github.io
+      User-Agent:
+      - Python-urllib/3.13
+    method: GET
+    uri: https://stac-extensions.github.io/datacube/v2.2.0/schema.json
+  response:
+    body:
+      string: "{\n  \"$schema\": \"http://json-schema.org/draft-07/schema#\",\n  \"$id\":
```

Comment:
Is there any way to load this content from another JSON file or have it directly embedded as JSON with newlines to improve readability?